src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/configuration/SwaggerConfig.java

```diff
@@ -26,8 +26,8 @@
 	@Value("${payment.service.api.version}")
 	private String version;
 	
-	@Value("${payment.service.environment}")
-	private String environment;
+	@Value("${payment.service.swagger.enabled}")
+	private String enabled;
```

Comment:
Can we use @value and bind it to a Boolean ?

Reply:
@alexjoybc - Good suggestion, but doesn't make things more efficient because the code I have in the present class to convert the string to a boolean has to go into the @Value declaration. For example: @Value("save us binding to a boolean value requires the same code that I've incorporated Can be done but the same code goes into the binding that I already have in the class. For example: @Value (#{Boolean.parseBoolean'${payment.service.swagger.enabled}')}). No native way to do this that I am aware of.